Security/src/RedisDataProtection/Program.cs

```diff
@@ -1,30 +1,34 @@
 using Microsoft.AspNetCore.DataProtection;
+using Steeltoe.Configuration.CloudFoundry;
 using Steeltoe.Connectors.Redis;
 using Steeltoe.Security.DataProtection.Redis;
 
 WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
 
+builder.AddCloudFoundryConfiguration();
+
 // Steeltoe: Setup Redis connector.
 builder.AddRedis();
 
-// Steeltoe: Setup data protection to use Redis connector.
-builder.Services.AddDataProtection().PersistKeysToRedis().SetApplicationName("redis-keystore");
-
 // Steeltoe: Enable session state.
-builder.Services.AddSession();
+builder.Services.AddSession(options =>
```

Comment:
Isn't this being handled in Steeltoe itself now?

Reply:
No, I put that [in the test](https://github.com/SteeltoeOSS/Steeltoe/blob/main/src/Security/test/DataProtection.Redis.Test/RedisDataProtectionBuilderExtensionsTest.cs#L47-L51).